src/model.py

```diff
@@ -104,6 +104,18 @@ def _auto_complete_inputs_and_outputs(auto_complete_model_config):
                 "dims": [1],
                 "optional": True,
             },
+            {
+                "name": "return_logprobs",
```

Comment:
Let's be consistent with TRTLLM and use return_log_probs for the input name: https://github.com/triton-inference-server/tensorrtllm_backend/blob/91c07d3acd865238176f0fcc3375155e3085c5c8/all_models/inflight_batcher_llm/ensemble/config.pbtxt#L442

Reply:
Should we also unify the output names?
https://github.com/triton-inference-server/tensorrtllm_backend/blob/91c07d3acd865238176f0fcc3375155e3085c5c8/all_models/inflight_batcher_llm/ensemble/config.pbtxt#L505-L512

The content of vLLM logprobs is different from TRT-LLM. It might not be a bad idea to keep them different to distinguish between their difference.